tests/modeltests/vector_static_gcc/vector_static_gcc_v9.1.0.actual.txt

```diff
@@ -76,7 +76,7 @@
 4294857  4188c9:	c5 f9 28 c1          	vmovapd %xmm1,%xmm0
 4294861  4188cd:	5b                   	pop    %rbx
 4294862  4188ce:	5d                   	pop    %rbp
-4294863  4188cf:	c3                   	retq   
```

Comment:
why are these things changed here? this sounds totally unrelated?

Reply:
it is. I don't now how it ended up here